fdbserver/masterserver.actor.cpp

```diff
@@ -1276,6 +1276,8 @@ ACTOR Future<Void> trackTlogRecovery( Reference<MasterData> self, Reference<Asyn
 			.detail("Status", RecoveryStatus::names[RecoveryStatus::fully_recovered])
 			.trackLatest("MasterRecoveryState");
 
+			TraceEvent("MasterRecoveryFullyRecovered").trackLatest("MasterRecoveryFullyRecovered");
+
```

Comment:
I think doing it this way could result in you getting an inaccurate time if you are in the middle of a recovery and there was a previous recovery using this process as the master. If you only want to target the most recent generation, this would be a problem, but even in the case that you do want to go back to the prior recovery, it may have occurred on a different master and this one could be too old.

Reply:
One solution could be to use the `MasterRecoveryState` event and checking the current status. An alternative may be to include the ID of the master that recovered in the track latest key so that you know you aren't getting another.

Reply:
Done. Now only tracking current generation.